starter/src/assets/script.js

```diff
@@ -1,11 +1,9 @@
-/* Create an array named products which you will use to add all of your product object literals that you create in the next step. */
-
 /* Create 3 or more product objects using object literal notation 
    Each product should include five properties
    - name: name of product (string)
    - price: price of product (number)
-   - quantity: quantity in cart should start at zero (number)
-   - productId: unique id for the product (number)
```

Comment:
productId works -- but SKU is a real-world term for the product Id. I'm not convinced this is a necessary or helpful change.

Reply:
Ok, but how about starting the quantity at 0 and changing integer to number?

Reply:
Yep. That works